packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form-side-note.tsx

```diff
@@ -25,119 +25,77 @@ const AccountTransferBullet = ({ children }: React.PropsWithChildren) => (
-    ]);
+        return (
+            <Localize i18n_default_text='No fees for transfer between fiat account (with the same currency) to Deriv MT5, and Deriv X account(s), vice versa.' />
+        );
+    }, [currency, minimum_fee, transfer_fee]);
 
-    const getDxtradeAllowedNotes = useCallback(() => {
-        if (is_dxtrade_allowed) {
+    const getPlatformsAllowedNotes = useCallback(() => {
+        if (is_ctrader_transfer || is_dxtrade_transfer) {
             return (
                 <React.Fragment>
                     <AccountTransferBullet>
                         <Localize
-                            i18n_default_text='You may transfer between your Deriv fiat, cryptocurrency, {{platform_name_mt5}}, {{platform_name_ctrader}}, and {{platform_name_dxtrade}} accounts.'
-                            values={{ platform_name_dxtrade, platform_name_mt5, platform_name_ctrader }}
+                            i18n_default_text='Each day you can transfer up to {{ allowed_dxtrade }} {{ currency }}. The daily limit will be reset at 00:00 GMT.'
+                            values={{
+                                allowed_dxtrade: (
+                                    exchange_rate * (Number(allowed_transfers_amount?.dxtrade) || 50000)
+                                ).toFixed(2),
+                                currency: getCurrencyDisplayCode(currency),
+                            }}
                         />
                     </AccountTransferBullet>
+                </React.Fragment>
+            );
+        } else if (is_mt_transfer) {
+            return (
+                <React.Fragment>
                     <AccountTransferBullet>
                         <Localize
-                            i18n_default_text='Each day, you can make up to {{ allowed_internal }} transfers between your Deriv accounts, up to {{ allowed_mt5 }} transfers between your Deriv and {{platform_name_mt5}} accounts, up to {{ allowed_ctrader }} transfers between your Deriv and {{platform_name_ctrader}} accounts, and up to {{ allowed_dxtrade }} transfers between your Deriv and {{platform_name_dxtrade}} accounts.'
+                            i18n_default_text='Each day you can transfer up to {{ allowed_mt5 }} {{ currency }}. The daily limit will be reset at 00:00 GMT.'
                             values={{
-                                allowed_internal: allowed_transfers_count?.internal,
-                                allowed_mt5: allowed_transfers_count?.mt5,
-                                allowed_dxtrade: allowed_transfers_count?.dxtrade,
-                                allowed_ctrader: allowed_transfers_count?.ctrader,
-                                platform_name_dxtrade,
-                                platform_name_mt5,
-                                platform_name_ctrader,
+                                allowed_mt5: (
+                                    exchange_rate * (Number(allowed_transfers_amount?.mt5) || 200000)
```

Comment:
@lubega-deriv 
could we avoid `magic numbers` like 200000, 100000, 50000.
could we create an object at the top of the file
e.g const ALLOWED_TRANSFER_AMOUNTS= {MT5: 200000, CTRADER: ...., etc.}

Reply:
Updated